packages/migration/src/migrations/hearth/20240923104723-change-task-input-output-id-with-postfix.ts

```diff
@@ -0,0 +1,112 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors located at https://github.com/opencrvs/opencrvs-core/blob/master/AUTHORS.
+ */
+
+import { Db, MongoClient } from 'mongodb'
+
+const ADDRESS_FIELDS_TO_MIGRATE = [
+  'countryPrimary',
```

Comment:
I based the list on what we have under `packages/client/src/tests/mock-drafts.ts`